### PR TITLE
doc/releases/nautilus: final notes for v14.2.0

### PR DESCRIPTION
**Pull request by liewegas**

### CODE REVIEW

File: doc/releases/nautilus.rst [hunk 1]
```diff
@@ -320,29 +315,34 @@ Instructions
    and verify that each monitor has both a ``v2:`` and ``v1:`` address
    listed.
 
-#. For each host that has been upgrade, you should update your
-   ``ceph.conf`` file so that it references both the v2 and v1
-   addresses.  Things will still work if only the v1 IP and port are
-   listed, but each CLI instantiation or daemon will need to reconnect
-   after learning the monitors real IPs, slowing things down a bit and
+#. For each host that has been upgraded, you should update your
+   ``ceph.conf`` file so that it either specifies no monitor port (if
```
Comment (tchaikov) [inline doc/releases/nautilus.rst]: might want to update https://github.com/ceph/ceph/blob/master/doc/cephfs/nfs.rst#configuration-for-libcephfs-clients to be in sync with this suggestion.

Comment (tchaikov) [reply doc/releases/nautilus.rst]: probably we can also update https://github.com/ceph/ceph/blob/master/doc/rados/configuration/mon-config-ref.rst#minimum-configuration to have an example using the new syntax, for instance
```
mon host =  [v2:192.168.2.8:40995,v1:192.168.2.8:40996] [v2:192.168.2.8:40997,v1:192.168.2.8:40998] [v2:192.168.2.8:40999,v1:192.168.2.8:41000]
```
and link it to `doc/rados/configuration/msgr2.rst` 's `Updating ceph.conf and mon_host` section.

also could link `doc/rados/configuration/network-config-ref.rst` to the same section.

File: doc/releases/nautilus.rst [hunk 2]
```diff
@@ -320,29 +315,34 @@ Instructions
...
+   still work if only the v1 IP and port are listed, but each CLI
+   instantiation or daemon will need to reconnect after learning the
+   monitors also speak the v2 protocol, slowing things down a bit and
    preventing a full transition to the v2 protocol.
 
    This is also a good time to fully transition any config options in
-   ceph.conf into the cluster's configuration database.  On each host,
-   you can use the following command to import any option into the
+   ``ceph.conf`` into the cluster's configuration database.  On each host,
+   you can use the following command to import any options into the
```
Comment (tchaikov) [inline doc/releases/nautilus.rst]: might want to link this note to `doc/rados/configuration/msgr2.rst` 's `Updating ceph.conf and mon_host` section as well. it features some examples,  so user can follow them.

**Comment by liewegas** [comment]
eventually.. i'll cherry-pick after this merges.

**Comment by theanalyst** [comment]
jenkins render docs

**Comment by smithfarm** [comment]
needs rebase?

**Comment by tchaikov** [comment]
@smithfarm why did you label this PR with needs-rebase? the changeset applies cleanly to master.

**Comment by liewegas** [comment]
It's confusing bc the pr is based on something in teh nautilus branch, but applies cleanly--ignore github's long list of commits in this case :)

**Pull request merged**